module-3/web-app/src/components/Pages/Document/Document.module.scss

```diff
@@ -1,5 +1,7 @@
 .editorContainer {
     width: 100%;
+    height: 100%;
+    min-height: 100vh;
```

Comment:
Just wanted to double check, if you paste really long text in editor, will it grow to fit content appropriately?

Reply:
Checked. It works correctly.